systest/backup/common/utils.go

```diff
@@ -91,6 +105,257 @@ func CopyToLocalFs(t *testing.T) {
 	require.NoError(t, testutil.DockerCp(srcPath, copyBackupDir))
 }
 
+//***************************************New Adds
+
+func RemoveContentsOfPerticularDir(t *testing.T, dir string) {
```

Comment:
as I said, maybe we can just delete the whole dir instead of deleting one file at a time. There are already functions for that in Go stdlib

Reply:
Resolved.

I am now using os.RemoveAll() for deleting an entire directory.